src/main/java/com/rabbitmq/client/impl/recovery/AutorecoveringConnection.java

```diff
@@ -103,17 +103,13 @@ public AutorecoveringConnection(ConnectionParams params, FrameHandlerFactory f,
      */
     public void init() throws IOException, TimeoutException {
         this.delegate = this.cf.newConnection();
-        this.addAutomaticRecoveryListener();
-    }
-
-    public void start() throws IOException {
```

Comment:
This method existed for a reason. Are you 100% sure it is safe to remove it now, or a good idea to do so?